SPECS/python-jwt/python-jwt.spec

```diff
@@ -1,3 +1,4 @@
+# This package refers to PyJWT(https://github.com/jpadilla/pyjwt). Not to be confused with python-jwt(https://github.com/davedoesdev/python-jwt)
```

Comment:
you need to increment release number because spec has changed (even if it is only a comment)

Reply:
Updated. Thanks!